src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -14032,66 +14035,53 @@ namespace SurfaceGeometry {
             // Create new material
             state.dataMaterial->TotMaterials = state.dataMaterial->TotMaterials + 1;
             newAirMaterial = state.dataMaterial->TotMaterials;
-            auto *thisMaterial = new Material::MaterialChild;
+            auto *thisMaterial = new Material::MaterialGasMix;
             state.dataMaterial->Material.push_back(thisMaterial);
             state.dataHeatBal->NominalR.redimension(state.dataMaterial->TotMaterials);
             thisMaterial->Name = MatNameStAir;
             thisMaterial->group = Material::Group::WindowGas;
             thisMaterial->Roughness = Material::SurfaceRoughness::MediumRough;
             thisMaterial->Conductivity = 0.0;
             thisMaterial->Density = 0.0;
-            thisMaterial->IsoMoistCap = 0.0;
-            thisMaterial->Porosity = 0.0;
+            // thisMaterial->IsoMoistCap = 0.0;
+            // thisMaterial->Porosity = 0.0;
             thisMaterial->Resistance = 0.0;
             thisMaterial->SpecHeat = 0.0;
-            thisMaterial->ThermGradCoef = 0.0;
+            // thisMaterial->ThermGradCoef = 0.0;
             thisMaterial->Thickness = distance;
-            thisMaterial->VaporDiffus = 0.0;
-            thisMaterial->gasTypes = Material::GasType::Custom;
-            thisMaterial->GasCon = 0.0;
-            thisMaterial->GasVis = 0.0;
-            thisMaterial->GasCp = 0.0;
-            thisMaterial->GasWght = 0.0;
-            thisMaterial->GasFract = 0.0;
-            thisMaterial->gasTypes(1) = Material::GasType::Air;
-            thisMaterial->GlassSpectralDataPtr = 0;
-            thisMaterial->NumberOfGasesInMixture = 1;
-            thisMaterial->GasCon(1, 1) = 2.873e-3;
-            thisMaterial->GasCon(2, 1) = 7.760e-5;
-            thisMaterial->GasVis(1, 1) = 3.723e-6;
-            thisMaterial->GasVis(2, 1) = 4.940e-8;
-            thisMaterial->GasCp(1, 1) = 1002.737;
-            thisMaterial->GasCp(2, 1) = 1.2324e-2;
-            thisMaterial->GasWght(1) = 28.97;
-            thisMaterial->GasFract(1) = 1.0;
+            // thisMaterial->VaporDiffus = 0.0;
+            // thisMaterial->GlassSpectralDataPtr = 0;
+            thisMaterial->numGases = 1;
+            thisMaterial->gases[0] = Material::gases[(int)Material::GasType::Air];
```

Comment:
All of these default constants are housed in the constexpr default `Material::Gas` array.